applications/satellite-imagery-geotiff.ipynb

```diff
@@ -0,0 +1,479 @@
2irS3yySLYVYpEZqivE7NxUW2wypQsO/NYsXGEsRf27Ar1NMbXihurxo4Y4VIWClu42AicdAPsKstpSj2K2ZPLU8ksmm0haKYx7qkPfRs+ON+56M6Q/idjFth23v0JYP68+yWMa+4DWYIaAe4RQtyVOer+i+NS++tij8zPBK3MBxZlqpJeP15tCQ1CNnsxP3fQk/zeg59jMWixmcQ8uH6Oqu2xw3UZdhw2kpiyZbOWROzyfGbCNoFK2Yg6PLG2yFP1ZTbTmPUkZjWJ+MzyLAcqg7xv+iijfoFRN8fxVp1znTaTfpGZ9iaj+SLTrQZ52+Ha6ggH+ge5b+YEO8v9fGF2mpsqQ7SSmH7PpxGF1HJ57ti3j9ONdV594CCWNDL8tVyeKE05sVnHd11eN7aTK4v9dJKEW7wWc1HIoO2gtNkdHyhXuX9tkQP9gzhScs/8KVaSkAdX59FoNtKYjSRGY6q5e+dPkWpNLZNnOVpfoWjZDLs5LvO3BvEVy5Ac9/plpDDGVhYmUVnZ/CbUaYYqkzRS1VM6MMBijIBiNjhHJ0b2xCqbx5yaEaiUOVOpCJsYq5dQ0uwjrsUWjLeSyc+7QhpdMKvMQT9PWTdxVIs2jpGRUXVmo4CZsI2XNnqtr7wU5hzSRm8+hzNkvtoVQ6QEHB1i6QRPQNW2TDtMmnacJUQP6UfaJC/M7Kh7f1DE7LETiOYhbTMgDYz7wqb9vTghvqV/LxiPAnuEEDuFEC7wVuCe7je11hta65rWeofWegfwMPA/nVDgUlK5+EPme6TF7IHsfrctkvXUM3kOMjFBdEKDHL/4slcymi8RqZShXJ5QpczFMZ9fX2LIyXE288BYiKPeAgswmi9x28AYthD8m3/4GAO2y1ihxGKnxXihzFKnzacXzpBqzc58ic8vzmAJwXC+SJSmhGnKqdYGnTRholhm1C9w5469PLqxzJX9g7xl214sIfj8iePEKmU0b2C9O0v9FGyXY/UVUq05vbnOrr5+PjD9HGfCFq/bvpfj1Bh3fVIkw6LFXNRm0oGpQh8ANw1vo+L5NNOE22tjPNVY3UJcAV9YnKHfy/HMxmpPe2uy2Mdnz50l1YpAK5biDs00IdCGLCqBPsvpOW8avxOj9ttV/U2F8WUJddojGlpg3iOnykKKMeOCbAEvA6Y6QisD9SUjuWbseCttZFL+7a33WieQNrHiFeYSUyGFXf0x1SGvNiBtE9sDDDs5plybnkyLahuIcdoGu7p1TSVrWeKLtjYtWXWkkaRIpBB4KKy0ScVy8JM1DgctQJKKLfVqZI5YFphTHquyllXaEbYw9swXe1jCKCV8s9sLJRWtdQK8B/hH4Dngw1rrZ4UQvy6EuOtf8/wvJZWLOPq23bJVhQizMPQqFSGZUwZdZdoX5/fFDamtLOGZ9WVu7htk0POJtaZiu4zbmtv7R3CIGfdMT75iuzSSGE9KDq8t4Vs2D67OsytX5G++93XG/Q/BVKGPfi/Hbf0jrIcBlhA8sb7EvkqNmpdntrlBrFKqrkeUplQ9n5VOG0dITm+u40iL5xvrPLaxTM52eNXe/eQsh53lfn50yKGWzQiCJDHKxnHEeKHMneO7aaUJM60GxxtrtFWKlaxRFyVONtb5y5kzWELw0LmzPLo8x1yrQdV2WYpD8/jqPAthgNKaThpzqH+IsusRKsW5qEPZMdpVzzaNHlWojCaYrzskGM/1lkqYi0LSDEFWT+NsiG6gsmGWkH1pm1ZQJvGeyjwkdWKlWNNeJqtiOtda2JmXisLKlIa7Eiq9WVna3roOuhWqZTYb444DGNvesIsQzDYijg5NdZslCCdZNWgt4WYbj4junC51hqhbAwTCZTqKzHlkv0uoDipLrFttsg5Pxw5XeALsMpY2wqPHokw0EhhzfeOUmVVIHaWMzM9FHt+KSde3Elrr+7TWe7XWu7XWv5k99l6t9T3f4NiXvxhVClxKKhd3eJPZYpJVJ9oki65Q4LhosYgx6DKoMLnVDhMGOVRxcxxub3DfzAmmckXKUtHQLkInNJQkVCmeNLa0kVY0kpibhiYo2Q77+wZ4vLFKLHweWJ3HtywSNJN+kb9fOMWB6hBnNjeQQtLnuKRoIqXYW64a06himUQpcpbN3585xkqnzZH1ZXzbZj0MeHxhltONdXzb5rMzp/h8kOfExipD/pYk+lTfAAvtTX7t85/mH2dOoLRiPexwz/wpPrK0Tp/lcHj5HO+Y2Ea/7fKuqSsZ8gvcNbHbmIwtz1P1fL5vaJwj68ts8/J899gulqMOnTShartsxCG2kOws93OwWCEnJNu8vOGkyDwSerOTcStmLmzzYH0RV0hOd5qE2TrpZSrJQNYKkoQYDgjCZtKBqt4wFUGyBhlE+mu8Urpkx+4GoTtT6cqwCIm2TFWHXcm+p7BUxyDMsvZVaBml4Z7isHAz+K8Cp5YpGTd6LTdLJ1T0Jj4JO5zzzkG1QbVxVMtUX9LtzX2u9AugFatJgs7mQPs8I5svRWZlnNTRTg3cEapsEp/4wIv9KbngwuhKf/PbBUyov5RULu5QW1+7CDCdZGq1Zpc5zEbWF892qTrqJZa1JGI4X+TMZp3XbpviyU0j51FPIpoaytKgUGbCNs+0N9jvl9lMEwbdHEuhcQQ8UKzgCMHtA6Nm2IrgXBhwZf8QtVyeiaLxR5luNYjSlMsrNTxpsRC0WOq0DfvdcQmimMXNzZ4R17XVEW4en+SawVGOr6+yp3+ApaDFtbVRjqwvs6vcT8Fx6PdyuNKiGXQoux5BmnDFwDA52+EtNZ+WSnj3ZdeiGk9z6x//PvUkIk6NRlWQphysDrEZm9f7lm17+IuMy7LTLzKZL/F8sMm+Qh/zQROV2QqIbIFX2fzEE4Yl7Uubtigw6dpc2zfIUhwy7uV7/BmARmo0rdzsvfIEW4RFoCErrJI37yEKr+uP0k0m0JPi6Wl3ddnxmReJiFdMQsgqmFiWzDHnPeZlbdCoOzMRmSqwdGnrbKbijmxtVn
```

Comment:
In my case, computation ends, but plot never showed... Upon recomputation, notebook failed. Sorry I did not have time to investigate more....

Reply:
Thanks for these comments, I noticed this too. Usually for me, re-running the cell results in the plot showing, or rarely the kernel dies. 

I will debug this further today and see if I can get it to execute consistently.

Reply:
Fixed this by using `%matplotlib inline`